packages/arb-validator/cmd/rollupServer/rollupServer.go

```diff
@@ -159,6 +161,7 @@ func validateRollupChain() error {
 
 	// Rollup creation
 	auth := bind.NewKeyedTransactor(key)
+	auth.GasPrice = big.NewInt(int64(1e9*(*gasPrice))
```

Comment:
Approving with the note that this line is only safe since we can be confident that 1e9*gasPrice will be significantly less that uint64 max.

Reply:
Pushed commit that checks for overflow